src/FSLibrary/StellarKubeSpecs.fs

```diff
@@ -120,22 +120,16 @@ let NetworkDelayScriptResourceRequirements : V1ResourceRequirements =
 let GetSimulatePubnetResources networkSize : V1ResourceRequirements =
```

Comment:
should we just rename SimulatePubnetMixedLoadResourceRequirements and it in SimulatePubnet?

Reply:
I'll merge to PR to get the simulation unstuck, we can cleanup later.

Reply:
Done